ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -970,6 +970,13 @@ export default class ConfirmTransactionBase extends Component {
       isNoteToTraderSupported,
       ///: END:ONLY_INCLUDE_IN
     } = this.props;
+
+    if (!fromAddress) {
```

Comment:
This component is now choosing to render nothing given the properties it has, but should that decision instead be the responsibility of the parent component?

It would avoid unnecessarily rendering additional components, reduce the branches in each child component, plus add conceptual clarity as we can be confident if a component is rendered, it's going to render something.

Reply:
To clarify, I don't think this is applicable always, there's definitely exceptions if the condition was the result of lots of API calls and logic, or if the component always existed, but as we've no other early returns here, I'm just worried we're making a deeper issue higher up the tree.